DirectEmployers/MyJobs
# [PD-3225] page titles displaying line breaks

## What's this PR do?
Fixes the wrapping of the numbers inside of site-traffic analytics

## Screenshots (if appropriate)
https://directemployersfoundation.atlassian.net/browse/PD-3225
(3rd screenshot)

## Where should the reviewer start?
Start inside site-traffic analytics

## How should this be manually tested?
Pull down repository. You can verify that the visits column is not overlapping with extremely higher numbers now

## Any background context you want to provide?
Change per Jason Sole
